instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/__init__.py

```diff
@@ -102,6 +102,10 @@ def response_hook(span, instance, response):
     typing.Callable[[Span, redis.connection.Connection, Any], None]
 ]
 
```

Comment:
Could you add a section under `Usage` that outlines that async is supported as well as an example?

Reply:
Sure! Done 👍